### PR TITLE
operator namespace is now configurable through inputs.cue

### PR DESCRIPTION
**Pull request by alecholmez**
Signed-off-by: Alec Holmes <alecholmez@me.com>

### CODE REVIEW

File: inputs.cue [hunk 1]
```diff
@@ -14,6 +14,7 @@ config: {
 	debug:                       bool | *false @tag(debug,type=bool) // currently just controls k8s/outputs/operator.cue for debugging
 	test:                        bool | *false @tag(test,type=bool)  // currently just turns off GitOps so CI integration tests can manipulate directly
 	auto_copy_image_pull_secret: bool | *true  @tag(auto_copy_iamge_pull_secret, type=bool)
+	namespace: 					 string | *"gm-operator"
```
Comment (jack-bischoff) [inline inputs.cue:17]: Spacing

**Review by jack-bischoff — CHANGES_REQUESTED** [review]
1 style fix otherwise good

File: inputs.cue (hunk 1 above)
Comment (danielpcox) [inline inputs.cue:17]: I don't like the key name "namespace" for this. There are several relevant namespaces. Can we change it to something less generic, like "operator_namespace"?

Comment (alecholmez) [reply inputs.cue:17]: Sure thing I can change that.

Comment (tilleryd) [inline inputs.cue:17]: Do we want to follow the pattern of other fields here? I.e., be overridden via `@tag()`? Also, a comment would be good like `Change this to install the operator into a different namespace.`

Comment (alecholmez) [reply inputs.cue:17]: You got it, and yea the tag is actually a good idea will add that as well

**Comment by tilleryd** [comment]
@danielpcox Should this be added to https://github.com/greymatter-io/gitops-core/blob/main/k8s/outputs/operator.cue#L367-L373?

**Comment by danielpcox** [comment]
That file is currently only used by the integration tests. So if we want to swap the operator namespace in the integration tests, yes. Otherwise no point.

**Comment by tilleryd** [comment]
Cool, then I will approve this.

**Review by tilleryd — APPROVED** [review]

**Review by jack-bischoff — APPROVED** [review]

**Pull request merged**